ml/Config.cc

```diff
@@ -28,13 +28,13 @@ void ml_config_load(ml_config_t *cfg) {
     unsigned max_train_samples = config_get_number(config_section_ml, "maximum num samples to train", 4 * 3600);
     unsigned min_train_samples = config_get_number(config_section_ml, "minimum num samples to train", 1 * 900);
     unsigned train_every = config_get_number(config_section_ml, "train every", 1 * 3600);
-    unsigned num_models_to_use = config_get_number(config_section_ml, "number of models per dimension", 1);
+    unsigned num_models_to_use = config_get_number(config_section_ml, "number of models per dimension", 4);
```

Comment:
@vkalintiris i was hoping to leave all defaults alone and then change them later in here (once enough dogfooding done): https://github.com/netdata/netdata/pull/14222

Do we need to have `4` here or was it just better to have to make it easier for testing this PR etc?

Reply:
Just for testing the PR. I'll change it back to 1.